content/pt-br/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Agentes de execução do Contêiner
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+{{< note >}}
+Dockershim foi removido do projeto Kubernetes desde a versão 1.24. Para mais detalhes leia [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq).
+{{< /note >}}
+
+Você precisa instalar um {{< glossary_tooltip text="agente de execução do contêiner" length="all"  term_id="container-runtime" >}} em cada nó do cluster para que os pods possam ser executados. Esta página descreve as tarefas e o que está envolvido para configurar os nós. 
+
+O Kubernetes 1.24 requer um tempo de execução que esteja em conformidade com {{< glossary_tooltip term_id="cri" length="all" text="Container Runtime Interface">}} (CRI).
+
+Para mais informações, veja [versão de suporte CRI](#cri-versions).
+
+Esta página fornece uma descrição de como utilizar múltiplos container runtime comuns com Kubernetes.
+
+- [containerd](#containerd)
+- [CRI-O](#cri-o)
+- [Docker Engine](#docker)
+- [Mirantis Container Runtime](#mcr)
+
+{{< note >}}
+As versões do Kubernetes anteriores ao v1.24 incluem uma integração direta com o Docker Engine, utilizando um componente chamado _dockershim_. Esta integração direta especial não faz mais parte do Kubernetes (a remoção foi anunciada como parte da versão v1.20). Leia [Check whether Dockershim removal affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/) para entender como esta remoção pode afetar você. Para aprender mais sobre migração utilizando docker, acesse o [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
```

Comment:
You can translate `[Check whether Dockershim removal affects you]` and `[Migrating from dockershim]`

Reply:
Do you think that:

- Verifique se a remoção do Dockershim prejudica
- Migrando do dockershim

would be good translation?

Reply:
Maybe "Verifique se a remoção do Dockershim afeta sua configuração" (or "seu cluster", or even "afeta você")?

"Migrando do dockershim" sounds good to me.